src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -177,6 +177,7 @@ class TezosPlatformDiscoveryOperations(
         } else {
           (for {
             _ <- caching.putEntities(networkName, ent)
+            _ <- IO.contextShift(executionContext).shift
```

Comment:
If it helps, you can reduce repetition by making an implicit ContextShift[IO] available at the class beginning, and simply call `IO.shift` henceforth

Reply:
ok, done